src/components/NetworkConnectivity.js

```diff
@@ -105,7 +105,7 @@ class NetworkConnectivity extends React.PureComponent<Props, State> {
 
   componentWillUnmount() {
     const handler = this.getConnectionChangeHandler();
-    NetInfo.isConnected.removeEventListener('connectionChange', handler);
+    NetInfo.removeEventListener('connectionChange', handler);
```

Comment:
Here we should call the unsubscribe function returned by `NetInfo.addEventListener(fn)`